pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/nonpepperkit/NonPepperStatusKitService.java

```diff
@@ -49,6 +58,89 @@ public static String convertTimeStringIntoTimeStamp(Long dateMillis) {
         return instant.toString();
     }
 
+    public static String calculateCurrentStatus(ResultSet foundKitResults) {
```

Comment:
This method calculates the current status of a kit based on the logic that kit pages use in DSM for choosing which kits to show as without label, queue, sent, etc

Reply:
Moved! Thanks for noticing this!